include/dba.php

```diff
@@ -863,7 +863,7 @@ private static function buildRelationData() {
 	 *
 	 * @return boolean|array was the delete successful? When $in_process is set: deletion data
 	 */
-	public static function delete($table, array $conditions, $in_process = false, array &$callstack = [])
+	public static function delete($table, array $conditions, array $options = [], $in_process = false, array &$callstack = [])
```

Comment:
Please add the `$options` parameter to the doxygen header

Reply:
Good catch.